src/components/DataTable.js

```diff
@@ -200,7 +201,10 @@ export default ajaxCaller(class DataTable extends Component {
                         }
                       }, [
                         h(Sortable, { sort, field: name, onSort: v => this.setState({ sort: v }) }, [
-                          h(HeaderCell, [name])
+                          !!columnNamespace && div({
+                            style: { fontStyle: 'italic', color: colors.dark(0.75), paddingRight: '0.2rem' }
+                          }, [columnNamespace]),
+                          h(HeaderCell, [columnName])
```

Comment:
If you have a long namespace, the text can overflow and bump into the resize handle. The `HeaderCell` handles overflow properly, so we should put both pieces of text inside it.

Reply:
Makes sense! Is it important to maintain the short circuit from ` !!columnNamespace &&` inside the headerCell? I wasn't sure how to make that work

Reply:
Yep, the `HeaderCell` can have multiple children, with one of them being conditional. (We'll probably want a `span` instead of a `div`, so the text flows properly)

Reply:
Ah, didn't realize `HeaderCell` could take children, that solves it, thanks!